### PR TITLE
Animation and interaction fix

### PR DESCRIPTION
**Pull request by Adammui**
- Added skip for first cutscene with "Space" key;
- Fixed cat walking animation bugs (while drinking milk, with egg, with acorn);
- Aligning and checking all colliders and hints;
- Separate meow using key "M", added hint to it into the keybinds menu;
- Pavetnik now disappears after cutscene;

### CODE REVIEW

File: Fantasy-Quest/Assets/Scripts/UI/SlideshowParent.cs [hunk 1]
```diff
@@ -116,6 +137,7 @@ private IEnumerator ShowSlide(SlideStruct slide)
         [Button]
         public void StartSlideshow()
         {
+            playerInput.Player.Skip.performed += SkipCutscene;
```
Comment (Kiuh) [inline Fantasy-Quest/Assets/Scripts/UI/SlideshowParent.cs]: Do It in `OnEnable`

File: Fantasy-Quest/Assets/Scripts/Interaction/InteractionImpl.cs [hunk 2]
```diff
@@ -119,6 +106,40 @@ private IEnumerator InteractHumanRoutine()
             interactors.ForEach(x => x.InteractByHuman());
         }
 
+        private IEnumerator CatMeowingCoroutine()
```
Comment (Kiuh) [inline Fantasy-Quest/Assets/Scripts/Interaction/InteractionImpl.cs]:
```suggestion
        private IEnumerator CatMeowingRoutine()
```

File: Fantasy-Quest/Assets/Scripts/Interaction/InteractionImpl.cs [hunk 3]
```diff
@@ -119,6 +106,40 @@ private IEnumerator InteractHumanRoutine()
...
+                _ = catSkeleton.AnimationState.SetAnimation(0, canDoAnim, false);
+                canDoSound.PlayClip();
+                yield return new WaitForSeconds(canDoAnim.Animation.Duration);
+                _ = catSkeleton.AnimationState.SetAnimation(0, idleAnim, false);
+                canDoSmth = true;
+                LockerSettings.Instance.UnlockAll();
+            }
+        }
+
+        private IEnumerator CatHissingCoroutine()
```
Comment (Kiuh) [inline Fantasy-Quest/Assets/Scripts/Interaction/InteractionImpl.cs]:
```suggestion
        private IEnumerator CatHissingRoutine()
```

**Review by Kiuh — CHANGES_REQUESTED** [review]

File: Fantasy-Quest/Assets/Scripts/UI/SlideshowParent.cs (hunk 1 above)
Comment (Adammui) [reply Fantasy-Quest/Assets/Scripts/UI/SlideshowParent.cs]: Could i use `playerInput.Enable()` outside of  `OnEnable`,  ?

**Review by Kiuh — APPROVED** [review]

**Pull request merged**